### ISSUE TITLE
Accessibility Disclaimer Banner does not work

### ISSUE THREAD

**Issue by ruoqianz** [smlegitime/accessible-ui-helper#35]
![a](https://github.com/user-attachments/assets/d1115603-2896-465d-813e-36c0a205387f)
When I clicked read more and accept bottom, there is no response.
